.github/workflows/rolling_batch_integration.yml

```diff
@@ -0,0 +1,172 @@
+name: Rolling Batch Integration tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      djl-version:
+        description: 'The released version of DJL'
+        required: false
+        default: ''
+  schedule:
+    - cron: '0 15 * * *'
```

Comment:
let's schedule a different time, 3PM GMT is too busy.

Reply:
Done